src/app/InteractionModelEngine.h

```diff
@@ -102,17 +104,28 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
      *  Initialize the InteractionModel Engine.
      *
      *  @param[in]    apExchangeMgr    A pointer to the ExchangeManager object.
+     *  @param[in]    apFabricTable    A pointer to the FabricTable object.
+     *  @param[in]    apCASESessionMgr An optional pointer to a CASESessionManager (used for re-subscriptions).
      *
      *  @retval #CHIP_ERROR_INCORRECT_STATE If the state is not equal to
      *          kState_NotInitialized.
      *  @retval #CHIP_NO_ERROR On success.
      *
      */
-    CHIP_ERROR Init(Messaging::ExchangeManager * apExchangeMgr, FabricTable * apFabricTable);
+    CHIP_ERROR Init(Messaging::ExchangeManager * apExchangeMgr, FabricTable * apFabricTable,
+                    CASESessionManager * apCASESessionMgr = nullptr);
```

Comment:
This = nullptr default argument is very much appreciated as an obvious signal, in addition to the docs header, that null is legal.